metricbeat/module/nats/connection/connection_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"io/ioutil"
```

Comment:
So this can be removed?

Reply:
good catch :+1: sorry that I have missed this! It got removed with https://github.com/elastic/beats/pull/36963/commits/937a28b4c1df6af34f7e5d6d60fee3f5d56002b6